Makefile

```diff
@@ -120,19 +142,115 @@ check.lint: ; $(info $(H) running golint on packages...)
 
 # @todo gometalinter targets?
 
-build: setup
-	bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) //...
+build: setup go-build
+
+#-----------------------------------------------------------------------------
+# Target: go build
+#-----------------------------------------------------------------------------
+
+.PHONY: go-build
+
+.PHONY: pilot
+pilot: vendor
+	go install istio.io/istio/pilot/cmd/pilot-discovery
+
+.PHONY: pilot-agent
+pilot-agent: vendor
+	go install istio.io/istio/pilot/cmd/pilot-agent
+
+.PHONY: istioctl
+istioctl: vendor
+	go install istio.io/istio/pilot/cmd/istioctl
+
+.PHONY: sidecar-initializer
+sidecar-initializer: vendor
+	go install istio.io/istio/pilot/cmd/sidecar-initializer
+
+.PHONY: mixs
+mixs: vendor
```

Comment:
i'm not clear where this belongs, but at some point we need to add a `generate` target that can call `go generate istio.io/istio/...` before it does a full build.